vividus-plugin-azure-cosmos-db/src/main/java/org/vividus/azure/cosmos/CosmosDbSteps.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.azure.cosmos;
+
+import java.util.Set;
+import java.util.function.Function;
+
+import org.jbehave.core.annotations.When;
+import org.vividus.azure.cosmos.model.CosmosDbContainer;
+import org.vividus.bdd.context.BddVariableContext;
+import org.vividus.bdd.variable.VariableScope;
+import org.vividus.softassert.SoftAssert;
+import org.vividus.util.property.PropertyMappedCollection;
+
+public class CosmosDbSteps
+{
+    private static final int CREATED = 201;
+    private static final int NO_CONTENT = 204;
+    private static final int OK = 200;
```

Comment:
is it possible to use existing constants from transitive HTTP libs?

Reply:
was not able to find anything useful, won't add extra dependency just to refer status codes